lib/preview-web/src/PreviewWeb.test.ts

```diff
@@ -1067,7 +1067,8 @@ describe('PreviewWeb', () => {
 
     it('resets a single arg', async () => {
       document.location.search = '?id=component-one--a';
-      await createAndRenderPreview();
+      const preview = await createAndRenderPreview();
+      const onUpdateArgsSpy = jest.spyOn(preview, 'onUpdateArgs');
```

Comment:
I added expects for the call to`onUpdateArgs`. To me, it made it easier to understand what was going inside of `onResetArgs`. Please let me know if this is appropriate and what you think of the testing in general 🙂

Reply:
I guess I just check the `STORY_ARGS_UPDATED` event as basically the same thing as checking `onUpdateArgs`, as that is the only place it is emitted. Probably the spy is a little more coupled to the implementation than strictly necessary but I'm happy to leave it.